prime-router/docs/proposals/0017-serializer-error-codes.md

```diff
@@ -43,23 +48,33 @@ that would be helpful for the front-end, such as the specific field in question,
 The front-end can then use the errorCode and other metadata in the messages to calculate and display user-friendly messages.
 
 The following is the definition of the new action message that should be added. Its usage will be discussed in the
-sections below.
+sections below. Please note, `FieldProcessingMessage` may not look exactly like what is shown below. For example, some of
+its fields may be factored out and placed in a parent class, like `ItemActionLogDetail`. The other ActionMessages in
+`ActionMessages.kt` will most likely end up inheriting from `FieldProcessingMessage`. Currently, the main purpose of the
+would-be children of `FieldProcessingMessage` is to override the message parameter with a pre-built message specific
+to a particular normalization error. See `MissingFieldMessage` for an example.
 
 ```kotlin
+// This is a pseudocode class posted here just to show what kind of fields it would have. 
+// During implementation, this class most likely will have some fields factored out and 
+// integrated into ItemActionLogDetail. Other ActionMessages will probably get refactored as well
 class FieldProcessingMessage(
-    // Name of the HL7 field
-    fieldMapping: String = "",
+    // Name of the HL7 field (example: PID-29)
+    fieldId: String = "",
+    // Long name of the fieldName (example: patient time of death)
+    fieldName: String = "",
     // the ReportStream Element type associated with fieldMapping 
     // for the particular schema associated
-    fieldType: String = "",
+    fieldType: Element.Type = Element.Type.UNKOWN,
```

Comment:
Minor typo here: `UNKNOWN`

Reply:
I swear I caught that /facepalm. will fix.